wally-registry-backend/Cargo.toml

```diff
@@ -7,6 +7,10 @@ publish = false
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[features]
+default = ["s3-storage"]
```

Comment:
I don't think we should include this as a default. Defaults can be spooky as removing them later can be semver incompatible and I feel like it somewhat defeats the point of this being a feature flag. The rest of the PR looks great though!